eddies/github-webhook-cloud-function
# Trello id at end of branch names

**Trello id at end of branch names**
* * *

# What does this Pull Request do?
Closes #20 

Looks for the Trello Card ID/shortLink at the end of the branch name with a
`#` separator character. For example, given a Trello Card ID of 'nqPiDKmw', the
branch name should now resemble `my-branch-name#nqPiDKmw`.

# What's new?
This is a breaking change. Previously, the Trello card ids were at the beginning of the branch name, separated by a `/`, e.g. `nqPiDKmw/my-branch-name`.

# How should this be tested?

* Create a Trello card
* Create a new branch with the Trello card at the end of the branch name
* Create and push a commit on the branch
* Verify that the webhook fires and updates the Trello card

# Checklist
